spec/lib/multi_kafka_producer_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 shared_examples "default adapter" do
   it 'should set the default adapter for the platform' do
+    allow(Rails.env).to receive(:test?).and_return(false)
     MultiKafkaProducer.adapter =  nil
```

Comment:
not sure i'm following this, if it's set to nil before the let block runs won't it just be nil in the expectation?